mvlearn/embed/mvmds.py

```diff
@@ -49,6 +49,45 @@ class MVMDS(BaseEmbed):
 
     Notes
     -----
+    
+    Classical Multiview Multidimensional Scaling can be broken down into two
+    steps. The first step involves calculating the Euclidean Distance matrices, 
+    :math:`X_i`, for each of the :math:`k` views and double-centering 
+    these matrices through the following calculations:
+
+    .. math::
+        \Sigma_{i}=-\frac{1}{2}J_iX_iJ_i
+
+    .. math::
+        \text{where }J_i=I_i-{\frac {1}{n}}11_i'
```

Comment:
I think `11_i`  is a typo? you want `\mathbb{1}\mathbb{1}^T` I think. Also the choice of X_i is a bit misleading since the ith view data matrix we usually represent as X_i

Reply:
This is the same documentation as the following: https://en.wikipedia.org/wiki/Multidimensional_scaling

Reply:
Will change the X_i